pkg/ebpf/bpf_client.go

```diff
@@ -621,6 +622,12 @@ func (l *bpfClient) loadBPFProgram(fileName string, direction string,
 
 	start := time.Now()
 	l.logger.Info("Load the eBPF program")
+
+	if strings.Contains(podIdentifier, ".") {
+		l.logger.Info("Pod identifier cannot contain '.'")
+		return nil, -1, errors.New("Pod identifier cannot contain '.'")
```

Comment:
`Pod identifier` is an internal variable name, so let's modify the error string to `Pod name cannot contain '.'` instead ...